src/component/EditNote/editNote.css

```diff
@@ -0,0 +1,33 @@
+.modal-container {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: var(--COLOR-WHITE);
+  padding: 3rem;
+  z-index: 500;
```

Comment:
suggestion: use css variables for z-indexs

Reply:
ok